### PR TITLE
Also pin github.com/openzipkin-contrib/zipkin-go-opentracing

### PR DESCRIPTION
**Pull request by cybertron**
This repo got moved on github, so now we need to specify a version
for the new location as well or we get a newer release that doesn't
work.

### 1. Why is this pull request needed and what does it do?
See commit message
### 2. Which issues (if any) are related?
None
### 3. Which documentation changes (if any) need to be made?
None
### 4. Does this introduce a backward incompatible change or deprecation?
No

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -19,6 +19,7 @@ require (
 	github.com/miekg/dns v1.1.14
 	github.com/openshift/coredns-mdns v0.0.0-20190710141149-66c708637267
 	github.com/opentracing/opentracing-go v1.1.0
+	github.com/openzipkin-contrib/zipkin-go-opentracing v0.3.5
 	github.com/openzipkin/zipkin-go-opentracing v0.3.5
```
Comment (danehans) [inline go.mod]: @cybertron is `github.com/openzipkin/zipkin-go-opentracing` still required?

Comment (cybertron) [reply go.mod]: I can get rid of it if I change the import to reference the new location. I'll add that change to the PR.

**Comment by danehans** [comment]
@cybertron can you help me understand why this is needed when the trace plugin is not [enabled](https://github.com/openshift/cluster-dns-operator/blob/master/assets/dns/configmap.yaml) and customizing the Corefile is not supported (as of now)?

**Comment by danehans** [comment]
@ironcladlou @Miciah instead of further diverging from upstream by merging this PR, can we pull upstream changes and provide a new openshift/coredns release? Upstream already contains the updated `openzipkin-contrib` pkg reference.

**Comment by cybertron** [comment]
I ran into this problem when I was trying to add a ci job on the coredns-mdns repo to make sure that changes there don't break compilation of coredns. At the time I didn't realize that the Dockerfile uses a different build command than the makefile, which means that the image build still works.

I think you're probably right that rebasing would be better than this. I'll see how involved that would be.

**Comment by ironcladlou** [comment]
@danehans yes, we should track upstream closely and carry the absolute minimum patches to integrate w/ the CI system and handle plugin enablement. So if an upstream commit has something we need, let's merge in the latest from upstream and do a new release.

**Comment by danehans** [comment]
@cybertron can you rebase on https://github.com/coredns/coredns/commit/b139ba34f370a4937bf76e7cc259a26f1394a91d instead of openshift/coredns diverging further from upstream?

**Comment by cybertron** [comment]
This won't be needed once we do the rebase to the last upstream coredns.

**Pull request closed without merging**